.travis.yml

```diff
@@ -26,14 +26,17 @@ matrix:
       os: windows
       language: shell # 'language: python' is an error on Travis CI Windows
       before_install:
-        - choco install python --version 3.7.5
```

Comment:
We should add a new build for python 3.9 under windows.

Choco has a package for 3.9.0b1. https://chocolatey.org/packages/python